python/src/spark_rapids_ml/umap.py

```diff
@@ -378,6 +409,10 @@ def _fit(self, dataset: DataFrame) -> "UMAPModel":
             dtype=type(raw_data[0][0]).__name__,
         )
 
+        print(
+            type(raw_data[0][0]).__name__,
+        )
```

Comment:
Is this still needed?  If so, do we want a more descriptive message in the logs?

Reply:
oops, leftover from debugging.